src/utils/helper-functions/versions.ts

```diff
@@ -0,0 +1,23 @@
+import * as semver from 'semver'
+
+let ngrxEffectsVersion: string | undefined
+function getNgrxEffectsVersion(): string {
+  if (ngrxEffectsVersion) return ngrxEffectsVersion
+
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const effects = require('@ngrx/effects/schematics-core') as {
+      platformVersion: string
+    }
+    // version in file is ^x.x.x, but this isn't valid for semver
```

Comment:
Really? I was pretty sure that it worked without any problems...

Reply:
Yea, it is - the ngrx version is scripted and the caret will be there. 
At least for now 😅, I'll add a check for it to be sure.